### PR TITLE
Config from cloud, Mysql support

### PR DESCRIPTION
**Pull request by farrukhshahzad**
I have AB changes FROM REVERTED PR also. Please check

### CODE REVIEW

**Comment by farrukhshahzad** [comment]
Please pull from master after this is merged to have clean code.

File: common/utils.py [hunk 1]
```diff
@@ -350,46 +394,56 @@ def get_item_topics(param):
...
-                hier_level = {}
-                hier_level[0] = curr_tree
-                if curr_levels:
-                    for i in range(1, level_list[0]+2):
-                        hier_level[i] = curr_levels.get(str(i-1), "Unkown")
-                    hier_level[level_list[0]+2] = curr_topic
-                else:
-                    hier_level[1] = curr_topic
-
-                tree_dict[item].append(hier_level)
```
Comment (A-Baji) [inline common/utils.py:392]: This is the new code. Also on line 387 can you change `"Unknown"` to `get_keyword_value("unknown")`

Comment (farrukhshahzad) [reply common/utils.py:392]: it is included on line 423 onward. I will fix 

> Unknown

 thing

Comment (A-Baji) [reply common/utils.py:392]: The code from lines 423 are incorrect. I highlighted the differences here:
image

The things I put in red are what we need to keep.

Comment (farrukhshahzad) [reply common/utils.py:392]: All tests are passing. You can do that change and do a PR.

Comment (A-Baji) [reply common/utils.py:392]: ok

**Review by A-Baji — APPROVED** [review]

**Pull request merged**